src/components/MyAccount/RequestsTab/CancelButton.tsx

```diff
@@ -52,6 +52,32 @@ const CancelButton = ({
     }
   }
 
+  function failureModalProps(hold) {
+    return {
+      bodyContent: (
+        <Box className={styles.modalBody}>
+          <Text sx={{ marginLeft: "l", marginRight: "m" }}>
+            Your request for{" "}
+            <span style={{ fontWeight: "var(--nypl-fontWeights-medium)" }}>
+              {hold.title}
+            </span>{" "}
+            has not been canceled. Please try again.
+          </Text>
+        </Box>
+      ),
+      closeButtonLabel: "OK",
+      headingText: (
+        <Heading className={styles.modalHeading}>
+          <>
+            <Icon size="large" name="errorFilled" color="ui.error.primary" />
```

Comment:
not a fan of how the icon is the same for whether the cancel action has failed or succeeded (I guess bc cancel = bad even if that's what you're trying to do?) I think one of these should be changed to the info icon

Reply:
i don't think that cancelling a hold is negative in the sense of the design system. you should use the same icons as the other success/failure modals.